connexion/middleware/main.py

```diff
@@ -0,0 +1,65 @@
+import pathlib
+import typing as t
+
+from starlette.types import ASGIApp, Receive, Scope, Send
+
+
+class MissingMiddlewareError(Exception):
```

Comment:
What's the reason for creating this exception class?

Reply:
Legacy of a previous implementation :)
Will remove it.